package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "headquarters-node",
-  "version": "0.1.2",
+  "version": "0.2.0",
```

Comment:
That's where you're wrong: stuff changed.
For example,  any client that was using OAuth stopped working because the api no longer recognizes the groupbuddies.com emails.  So the users in your local database are probably considered invalid and a new user account will be created (this happened on the blog,  which is what motivated me to bump the minor version for the Ruby wrapper) 

You're right in the sense that it's not a change as the semver describes it,  but it's one that likely requires actions from the developer to get the new version working (I can tell you later what I had to do for the blog).  That for me is enough to justify a minor version bump

That said: I really don't care that much as to keep this discussion going,  so feel free to revert this if you want